config/routes.rb

```diff
@@ -3,4 +3,6 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  resources :messages
```

Comment:
- [Optional] Since you are building an api, I don't think it is  necessary to have used a resourceful routes. All the other actions are not necessary for this app. We just needs to retrieve the greeting and make use of them in the frontend